src/renderer/fonts/caching_shaper.rs

```diff
@@ -42,8 +42,16 @@ impl CachingShaper {
         let font_key = self
             .options
             .as_ref()
-            .map(|options| options.fallback_list.first().unwrap().clone().into())
-            .unwrap_or(FontKey::Default);
+            .map(|options| FontKey {
+                italic: options.italic,
```

Comment:
Could `TryFrom` trait be used here?

Reply:
I guess it could, but it would only be used here and so I'm not sure it's worth the added complexity (and moving the logic out of this file thus breaking things up that could stay together).

Reply:
I agree with smolck. This is explicit about whats happening which I like